source/kraken/kraken_zmq.cpp

```diff
@@ -52,6 +55,27 @@ static void show_usage(const std::string& name, const boost::program_options::op
               << "\t KRAKEN_GENERAL_instance_name=<inst_name>" << std::endl;
 }
 
+void set_core_file_size_limit(int max_core_file_size, log4cplus::Logger& logger) {
+    if (max_core_file_size != 0) {
```

Comment:
```suggestion
    if (max_core_file_size > 0) {
```

Reply:
the reason why you might want to have a negative number is to set the max value to infinity.
```
# define RLIM_INFINITY ((__rlim_t) -1)
```

Reply:
ouchh oldies style